README.md

```diff
@@ -1,30 +1,26 @@
-# ECMAScript proposal: JavaScript standard library UUID module
+# ECMAScript proposal: JavaScript standard library UUID built-in
 
 Champions: [Benjamin Coe](https://github.com/bcoe)
 
 Status: Stage 0
```

Comment:
"Stage 0" is a bit ambiguous. Maybe say, "Status: early draft, never presented to TC39".